src/sentry/assistant/guides.py

```diff
@@ -22,4 +22,5 @@
     "for_review_guide": {"id": 9, "required_targets": ["for_review_guide_tab"]},
     "alerts_write_member": {"id": 10, "required_targets": ["alerts_write_member"]},
     "alerts_write_owner": {"id": 11, "required_targets": ["alerts_write_owner"]},
+    "release_adoption": {"id": 12, "required_targets": ["release_adoption"]},
```

Comment:
i got my pr in first, make sure to use id 13

Reply:
will do, thanks for the heads up 👍